### PR TITLE
Refactor .buildbot.sh and add morello-purecap support

### PR DESCRIPTION
**Pull request by drpdr**
Now we have an updated toolchain and the board:
- Address/close Issue #28

### CODE REVIEW

**Comment by drpdr** [comment]
bors try

File: .buildbot.sh [hunk 1]
```diff
@@ -2,12 +2,12 @@
 # Buildbot script to check that `split-llvm-extract`:
 #   1. Works under linux when applied to sqlite3 (proof of concept);
 #   2. works when applied to `lua` v5.4.0 cross compiling to CHERI.
-# Note: only RISCV64 is tested at the moment.
+# Since June 2022 we only support morello-purecap
```
[the remark this answers is not in the document]
Comment (drpdr) [reply .buildbot.sh:5]: Sure, we will support both.

**Comment by drpdr** [comment]
bors try

**Comment by drpdr** [comment]
bors try

**Comment by drpdr** [comment]
bors try

**Comment by drpdr** [comment]
bors try

**Comment by drpdr** [comment]
bors try

**Comment by drpdr** [comment]
bors try

**Comment by drpdr** [comment]
bors try

**Comment by drpdr** [comment]
bors try

**Comment by drpdr** [comment]
bors try

**Comment by drpdr** [comment]
A bit of a mess here but nothing major since we have to squash those experiments eventually.

**Comment by drpdr** [comment]
bors try

File: .buildbot.sh [hunk 2]
```diff
@@ -108,7 +126,7 @@ rm temp.bc
 rm lua.bc
 
 cd out-lua
-	make CC=$CC CFLAGS="-DLUA_USE_READLINE_DL -D__LP64__=1 $CONFIG_FLAGS" LDFLAGS="$CONFIG_FLAGS" LLVM_CONFIG=$CHERI/bin/llvm-config
+	make CC=$CC CFLAGS="-DLUA_USE_READLINE_DL -D__LP64__=1 $CONFIG_FLAGS" LDFLAGS="$CONFIG_FLAGS" LLVM_CONFIG=$CHERI/bin/llvm-config
```
Comment (ltratt) [inline .buildbot.sh:129]: Extraneous trailing space.

Comment (drpdr) [reply .buildbot.sh]: Fix in [6ba25b4](https://github.com/capablevms/llvm-function-split/pull/29/commits/6ba25b47556fdd516172a09f1c0bfdb8f02ce37f).

**Comment by ltratt** [comment]
I guess this is ready for squashing? If so, please do so.

**Comment by ltratt** [comment]
We still seem to have a merge commit?

**Comment by drpdr** [comment]
> We still seem to have a merge commit?

Trying to get rid of it...

**Comment by drpdr** [comment]
Hang on, let me reconstruct the order of the history first.

**Comment by drpdr** [comment]
If didn't mess up again, this should be now properly squashed @ltratt. I had a local branch issue.

**Pull request merged**